alphastats/DataSet.py

```diff
@@ -249,44 +266,15 @@ def load_metadata(self, file_path: Union[pd.DataFrame, str]) -> pd.DataFrame:
         elif file_path.endswith(".csv"):
             df = pd.read_csv(file_path)
         else:
-            df = None
             logging.warning(
                 "WARNING: Metadata could not be read. \nMetadata has to be a .xslx, .tsv, .csv or .txt file"
             )
-            return
+            return None
+
         if df is not None and self.sample not in df.columns:
             logging.error(f"sample_column: {self.sample} not found in {file_path}")
 
         # check whether sample labeling matches protein data
         #  warnings.warn("WARNING: Sample names do not match sample labelling in protein data")
         df.columns = df.columns.astype(str)
         return df
-
-    def _save_dataset_info(self):
-        n_proteingroups = self.mat.shape[1]
-        preprocessing_dict = {
-            PreprocessingStateKeys.RAW_DATA_NUM_PG: n_proteingroups,
-            PreprocessingStateKeys.NUM_PG: self.mat.shape[1],
-            PreprocessingStateKeys.NUM_SAMPLES: self.mat.shape[0],
-            PreprocessingStateKeys.INTENSITY_COLUMN: self.intensity_column,
-            PreprocessingStateKeys.LOG2_TRANSFORMED: False,
-            PreprocessingStateKeys.NORMALIZATION: None,
-            PreprocessingStateKeys.IMPUTATION: None,
-            PreprocessingStateKeys.CONTAMINATIONS_REMOVED: False,
-            PreprocessingStateKeys.CONTAMINATION_COLUMNS: self.filter_columns,
-            PreprocessingStateKeys.NUM_REMOVED_PG_DUE_TO_CONTAMINATION: 0,
-            PreprocessingStateKeys.DATA_COMPLETENESS_CUTOFF: 0,
-            PreprocessingStateKeys.NUM_PG_REMOVED_DUE_TO_DATA_COMPLETENESS_CUTOFF: 0,
-            PreprocessingStateKeys.MISSING_VALUES_REMOVED: False,
-        }
-        return preprocessing_dict
-
-    def overview(self):
```

Comment:
I would not delete this. However, I would rename it to __repr__ to follow conventions and add a todo to make this more useful.

Reply:
hm, any ideas how to make this more useful?